pkg/converter/converter.go

```diff
@@ -116,6 +116,8 @@ func (cvt *Converter) Convert(ctx context.Context, source, target, cacheRef stri
 			if err != nil {
 				if errors.Is(err, ctrErrdefs.ErrNotFound) {
 					logger.Infof("cache %s not found", cacheRef)
+				} else if errors.Is(err, ctrErrdefs.ErrNotImplemented) {
+					logger.Warnf("unsupported cache version")
```

Comment:
logger.Warnf("ignore cache %s, unmatched version: %s, expected: %s")

Reply:
We can log in `cache.Fetch`